data/settings.js

```diff
@@ -1068,13 +1220,13 @@ Value format: \`<header-field> [ ":" <regexp> ]\`
 Some spam scanners include the maximum score value in one of their status
 headers. Using this setting, this maximum can be extracted from the message
 itself instead of specifying the maximum manually using the setting
-[[setting,sieve_spamtest_max_value]].
+[[setting,sieve_spamtest_score_max_value]].
 
 The syntax is identical to the [[setting,sieve_spamtest_status_header]]
 setting.`
 	},
 
-	sieve_spamtest_max_value: {
+	sieve_spamtest_score_max_value: {
```

Comment:
value type is wrong, it's not UINT. Should explain it can have decimal point. Should also link the max_value and max_header settings together, and say in both that only one of them can be used at a time. Same for sieve_virustest

Reply:
fixed